test/verify/storagelib.py

```diff
@@ -172,11 +172,11 @@ def teardown():
 
     def content_tab_info_row(self, row_index, tab_index, title):
```

Comment:
Let's rename this this to `content_tab_info_label` since it no longer refers to an element that includes the whole row.

Reply:
fixed